.zenodo.json

```diff
@@ -1,14 +1,14 @@
 {
   "creators": [
     {
-      "affiliation": "Universitat Politècnica de Catalunya",
-      "name": "Manuel Molano Mazon",
-      "orcid": "0000-0002-9140-3571"
+      "affiliation": "Netherlands eScience Center, Amsterdam, The Netherlands",
+      "name": "Giulia Crocioni",
+      "orcid": "0000-0002-0823-0121"
```

Comment:
It's not very important, but slightly neater if we would make the listing of the affiliations uniform. 
Right now Robert and Manuel have no city or country at all, Jorge lists "NL", and eSC lists "The Netherlands".